tests/organize/test_models.py

```diff
@@ -43,7 +43,10 @@ def test_deploy_event_from_previous_event(get_or_create_gmail, base_application,
 
```

Comment:
This was caused by pre-commit (I initially added the test to this file, then realized it would belong to the `test_utils` module).